README.md

```diff
@@ -38,21 +38,28 @@ or using pip+git for the latest version of the code :
 pip install git+https://github.com/pyRiemann/pyRiemann
 ```
 
-Anaconda is not currently supported, if you want to use anaconda, you need to create a virtual environment in anaconda,
-activate it and use the above command to install it.
+#### Using conda
+
+The package is distributed via [conda-forge](https://conda-forge.org).
+We recommend installing it into a new, dedicated environment.
```

Comment:
why a new env? then you end up with one env per package and it makes no sense.

I would say "to activate the environment" you are willing to update if it's not your base environment and then just conda install

my 2c